CRM/Core/Smarty/plugins/block.serialize.php

```diff
@@ -1,37 +0,0 @@
-<?php
-/*
- +--------------------------------------------------------------------+
- | Copyright CiviCRM LLC. All rights reserved.                        |
- |                                                                    |
- | This work is published under the GNU AGPLv3 license with some      |
- | permitted exceptions and without any warranty. For full license    |
- | and copyright information, see https://civicrm.org/licensing       |
- +--------------------------------------------------------------------+
- */
-
-/**
- * CiviCRM's Smarty gettext plugin
- *
- * @package CRM
- * @author Donald Lobo <lobo@civicrm.org>
- * @copyright CiviCRM LLC https://civicrm.org/licensing
- */
-
-/**
- * Smarty block function providing serialization support
- *
- * See CRM_Core_I18n class documentation for details.
- *
- * @param array $params
- *   Template call's parameters.
- * @param string $text
- *   {serialize} block contents from the template.
- * @param CRM_Core_Smarty $smarty
- *   The Smarty object.
- *
- * @return string
- *   the string, translated by gettext
- */
-function smarty_block_serialize($params, $text, &$smarty) {
-  return serialize($text);
```

Comment:
This looks like the sort of thing that might have been used by schema `tpl.sql` files.

Reply:
That would make sense. Can keep it if it is needed, but I can't find any trace of it.